enso_driver.py

```diff
@@ -0,0 +1,37 @@
+import os
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.run import runner
+
+# Generic parameters. See enso.cfg for specific parameters.
+# Run with `python enso_driver.py -d enso.cfg`.
+param = CoreParameter()
+machine_path_prefix = '/global/project/projectdirs/acme/acme_diags'
+param.reference_data_path = os.path.join(machine_path_prefix, 'obs_for_e3sm_diags/time-series')
+param.test_data_path = os.path.join(machine_path_prefix, 'test_model_data_for_acme_diags/time-series/E3SM_v1')
+html_prefix = '/global/project/projectdirs/acme/www/forsyth'
```

Comment:
This path is specific to me. Should we just leave a comment for users to change it to their `html_prefix`?

Reply:
`html_prefix` has a hardcoded path, which makes `results_dir` hardcoded as well. This isn't good, since as [seen in the examples](https://e3sm-project.github.io/e3sm_diags/docs/html/quickguides/quick-guide-compy.html), `results_dir` is something that users set when running the diags.